node/consensus/consenter.go

```diff
@@ -48,17 +48,17 @@ func (c *Consenter) SimulateStateTransition(prevState *state.State, requests [][
 		filteredControlEvents = append(filteredControlEvents, ce)
 	}
 
-	newState, fragments := prevState.Process(c.Logger, filteredControlEvents...)
+	newState, fragments, configRequest := prevState.Process(c.Logger, filteredControlEvents...)
 	batchAttestations := aggregateFragments(fragments)
 
-	return newState, batchAttestations
+	return newState, batchAttestations, configRequest
 }
 
 // Commit indexes BAs and updates the state.
 // Note that this must hold: Commit(controlEvents) with the same controlEvents is idempotent.
 // TODO revise the recovery from failure or shutdown, specifically the order of Commit and Append.
 func (c *Consenter) Commit(events [][]byte) {
-	state, batchAttestations := c.SimulateStateTransition(c.State, events)
+	state, batchAttestations, _ := c.SimulateStateTransition(c.State, events)
```

Comment:
Extract the config request and do a log.Info on it if it exists.
Log there it is not supported yet.